packages/frontend/src/pages/routes/Router.tsx

```diff
@@ -1,12 +1,26 @@
-import React, { useState } from "react";
-import AuthenticatedRoutes from "./AuthenticatedRoutes";
-import UnauthenticatedRoutes from "./UnauthenticatedRoutes";
+import React, { useEffect, useState } from 'react';
+import AuthenticatedRoutes from './AuthenticatedRoutes';
+import UnauthenticatedRoutes from './UnauthenticatedRoutes';
+import { getAuth } from 'firebase/auth';
+import { useAuth } from '../../hooks/useAuth';
```

Comment:
I think we probably should have useAuth encapsulate the logic of firebase auth, @frasermcc9 what do you think?

Reply:
The token could become expired so I didn't want to store it inside our app, just always direct from source